MovieTime/MovieTimeTests/MTMovieFeedViewModelTests.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  MTMovieFeedViewModelTests.swift
+//  MovieTimeTests
+//
+//  Created by Enrique Melgarejo on 09/01/18.
+//  Copyright © 2018 Choynowski. All rights reserved.
+//
+
+import XCTest
+@testable import MovieTime
+
+class MTMovieFeedViewModelTests: XCTestCase {
+
+    override func setUp() {
+        super.setUp()
+        // Put setup code here. This method is called before the invocation of each test method in the class.
+    }
+
+    override func tearDown() {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+        super.tearDown()
+    }
+
+    func testInitialization() {
+        let viewModel = MTMoviesFeedViewModel()
+        XCTAssertNotNil(viewModel, "The view model should not be nil.")
```

Comment:
the initializer for `MTMoviesFeedViewModel` was declared as `init(service: MTMovieDBServiceProtocol = MTMovieDBService())` - what is the benefit of asserting it isn't nil?

Reply:
It is possible that there is a `Failable Initializer` in the *ViewModel*, so it is important to have this  unit test.